website_blog_excerpt_img/README.rst

```diff
@@ -0,0 +1,72 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Excerpt + Image in Blog
+=======================
+
+This module extends the functionality of blogs to support having a more
+attractive layout that displays an excerpt of each post, a "Read more" button
+and an image. All automatic.
```

Comment:
Hi @Yajo Thanks for another interesting contribution!
Please remove or rephrase "All Automatic"